src/status_im/accounts/update/publisher.cljs

```diff
@@ -0,0 +1,52 @@
+(ns status-im.accounts.update.publisher
+  (:require [re-frame.core :as re-frame]
+            [re-frame.db]
+            [status-im.constants :as constants]
+            [status-im.accounts.update.core :as accounts]
+            [status-im.pairing.core :as pairing]
+            [status-im.utils.async :as async-util]
+            [status-im.data-store.accounts :as accounts-store]
+            [status-im.utils.datetime :as datetime]
+            [status-im.transport.message.protocol :as protocol]
+            [status-im.transport.message.contact :as message.contact]
```

Comment:
`message.contact`, `protocol`,  `datetime`, `accounts-store`, `async-util`, `re-frame` are not used